cli/flags.rs

```diff
@@ -732,6 +733,17 @@ Generate html reports from lcov:
         .help("Output coverage report in lcov format")
         .takes_value(false),
     )
+    .arg(Arg::new("output")
+    .long("output")
+    .help("Output file (defaults to stdout) for lcov")
+    .long_help("Exports the coverage report in lcov format to the given file.
+Filename should be passed along with '='
+For example '--output=foo.lcov'
+If no --output arg is specified then the report is written to stdout."
+  )
+    .takes_value(true)
+    .require_equals(true)
```

Comment:
Do we really need to require equals sign here? Seems we could do the same as in `--lcov` and not require it

Reply:
- all the other args in the coverage like `include`, `exclude`, `regex` who accepts values all requires equals
- I think since the last arg `files` accepts multiple values it's better to be explicit about specifying

Reply:
Okay, makes sense, I was confused by the example shown above. Let's change the example to be less ambiguous